### DISCUSSION TITLE
Points of interest v3

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#151] [Q&A] [answered]
[I rewrote the POI format](https://github.com/mapsforge/mapsforge/pull/1411) for better **offline search** and more new features:

- Next Mapsforge POI updates will not be compatible with the current Cruiser 3.0.18

- OpenAndroMaps POI are still v2, so they will not be compatible with next Cruiser
(please download POI from Mapsforge)

- New Mapsforge POI require **Cruiser 3.0.19** which will be available in the next days

**Comment by devemux86** [comment, marked as answer]
Points of interest v3 are supported in [Cruiser 3.0.19](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

Please update downloaded POI from **Mapsforge**.

**Reply by DolomitesTom** [reply]
how can I manually upload it? I thougt it recognizes if there are updates and only then it can be updated.

**Reply by devemux86** [reply]
The updates of downloads are displayed in the 3rd screen of the download manager.

In the future we will have notifications for updates:
- https://github.com/devemux86/cruiser/discussions/86

<img src="https://github.com/devemux86/cruiser/assets/3484020/780629ed-0182-43b7-a04c-869795389144" width="50%"/>

**Reply by DolomitesTom** [reply]
ok, so no manual update needed, just wait till the new formats are available.

**Reply by devemux86** [reply]
We still need to press and start the download of the updates.

_POI v3 updates have a date of May 2023._
